make/Main.gmk

```diff
@@ -701,7 +701,7 @@ else
   endif
 
   # Building java.base-jmod requires all of hotspot to be built.
-  java.base-jmod: j9vm-build j9vm-compose-buildjvm 
+  java.base-jmod: j9vm-compose-buildjvm
```

Comment:
Can this be moved into closed?  It doesn't need to block this change, but that should be checked to decrease our direct changes to openjdk content.

Reply:
Either openj9 must define the `hotspot` make target or that line needs to change anyway.